src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task{
+
+    protected String dlDate;
```

Comment:
Variable name could be less ambiguous

Reply:
Like what Arindam mentioned, perhaps try to avoid using shortforms like "dl" to improve readability

Reply:
Got it, thanks guys!